antlr4/PostgreSQLParser.g4

```diff
@@ -1316,6 +1333,17 @@ for_clause
     : FOR ( UPDATE | NO KEY UPDATE | SHARE | KEY SHARE ) (OF table_name_ (COMMA table_name_)*)? ( NOWAIT | SKIP_ LOCKED)*
     ;
 
+updater_clause
+    : (expr EQUAL expr (COMMA expr EQUAL expr)*)
```

Comment:
Based on my reading of the grammar definition, I think we want the each of these three cases to be list-able. In other words, I think it should be possible to have `SET blah=1, (a, b, c) = (1, 2, 3), foo=2`. 

I think that's what you're alluding to in your comment to updater not being perfect. That's fine. We can iterate from here.